apps/studio/src/components/PageEditor/MenuBar.tsx

```diff
@@ -179,32 +222,32 @@ const MenuBar = ({ editor, schema }: { editor: Editor; schema: any }) => {
           icon: BiListOl,
           title: 'Ordered list',
           action: () => editor.chain().focus().toggleOrderedList().run(),
-          isActive: () => editor.isActive('orderedlist'),
+          isActive: () => editor.isActive('orderedList'),
         },
 
         {
           type: 'item',
           icon: BiListUl,
           title: 'Bullet list',
           action: () => editor.chain().focus().toggleBulletList().run(),
-          isActive: () => editor.isActive('unorderedlist'),
+          isActive: () => editor.isActive('bulletList'),
         },
       ],
     },
     {
       type: 'divider',
     },
     // {
-    //   type: "item",
+    //   type: 'item',
     //   icon: BiLink,
-    //   title: "Add link",
-    //   action: () => showModal("hyperlink"),
+    //   title: 'Add link',
+    //   action: () => showModal('hyperlink'),
     // },
     // {
-    //   type: "item",
+    //   type: 'item',
     //   icon: BiImageAdd,
-    //   title: "Add image",
-    //   action: () => showModal("images"),
+    //   title: 'Add image',
+    //   action: () => showModal('images'),
     // },
```

Comment:
we're not deleting cos we want to add images in later?

Reply:
Ah actually that wouldn't be using the tiptap editor, have removed some stuff that will be unused. Have only preserved the links + files which might be done on tiptap in future